deployments/liqo/values.yaml

```diff
@@ -74,19 +74,19 @@ networking:
       gateway:
         image:
           # -- Image repository for the gateway container.
-          name: "ghcr.io/liqotech/gateway"
+          name: "ghcr.io/cheina97/gateway"
```

Comment:
We need it to use the CI/CD images till the PR is merged. I will change it just before merging this PR